hypothesis-python/tests/cover/test_sampled_from.py

```diff
@@ -48,9 +50,13 @@ def test_can_sample_ordereddict_without_warning():
     sampled_from(an_ordereddict).example()
 
 
-@given(sampled_from(an_enum))
-def test_can_sample_enums(member):
-    assert isinstance(member, an_enum)
+@pytest.mark.parametrize("enum_class", [an_enum, a_flag, an_empty_flag])
+def test_can_sample_enums(enum_class):
+    @given(sampled_from(enum_class))
+    def test_it(member):
+        assert isinstance(member, enum_class)
+
+    test_it()
```

Comment:
I'd reach for our `assert_all_examples(sampled_from(enum_class), lambda x: isinstance(x, enum_class))` test helper here.